__init__.py

```diff
@@ -13,8 +13,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from TEAL.src import Amortization
-from TEAL.src import CashFlow_ExtMod
-from TEAL.src import CashFlows
-from TEAL.src import CashFlowUser
-from TEAL.src import main as CashFlow
+from .src import Amortization
+from .src import CashFlow_ExtMod
+from .src import CashFlows
+from .src import CashFlowUser
+from .src import main as CashFlow
```

Comment:
@joshua-cogliati-inl do we need to add the raven dir to the system path here so python knows about ravenframework?